src/main/resources/init_db.sql

```diff
@@ -12,3 +12,44 @@ ALTER TABLE `taxi_service`.`manufacturers`
 
 ALTER TABLE `taxi_service`.`manufacturers`
     RENAME COLUMN `delete` TO `deleted`;
+
+CREATE TABLE `taxi_service`.`drivers` (
+    `driver_id` BIGINT(11) NOT NULL AUTO_INCREMENT,
+    `driver_name` VARCHAR(225) NOT NULL,
```

Comment:
there is no need to indicate `driver_` prefix for each column because the columns are located in `drivers` table

same for other column names in manufacturer and car tables

Reply:
done. should I do it for `id` column as well?

Reply:
yep

Reply:
done